crates/hyperswitch_connectors/src/connectors/jpmorgan/transformers.rs

```diff
@@ -56,84 +138,249 @@ impl TryFrom<&JpmorganRouterData<&PaymentsAuthorizeRouterData>> for JpmorganPaym
     ) -> Result<Self, Self::Error> {
         match item.router_data.request.payment_method_data.clone() {
             PaymentMethodData::Card(req_card) => {
+                if item.router_data.is_three_ds() {
+                    return Err(errors::ConnectorError::NotSupported {
+                        message: "3DS payments".to_string(),
+                        connector: "Jpmorgan",
+                    }
+                    .into());
+                }
+
+                let capture_method =
+                    map_capture_method(item.router_data.request.capture_method.unwrap_or_default());
+
+                let currency = item.router_data.request.currency;
+
+                let merchant_software = JpmorganMerchantSoftware {
+                    company_name: String::from("JPMC"),
+                    product_name: String::from("Hyperswitch"),
+                };
+
+                let merchant = JpmorganMerchant { merchant_software };
+
+                let expiry: Expiry = Expiry {
+                    month: req_card.card_exp_month,
+                    year: req_card.card_exp_year,
```

Comment:
We need to pass 4 digit expiry year only. Use function `get_expiry_year_4_digit`

Reply:
alright